templates/spinnaker_application_manifest_middle_secured.yaml

```diff
@@ -15,3 +15,6 @@
 
       # Configuration backups
       Full backups are stored in [this repo](https://source.cloud.google.com/$PROJECT_ID/$CONFIG_CSR_REPO).
+
+      # Stay up to date
+      Join the [Spinnaker for GCP Announce](https://groups.google.com/forum/#!forum/spinnaker-for-gcp-announce) Google Group.
```

Comment:
@dorbin Thoughts on "Sign up for Spinnaker for GCP updates and annoucements" or so?